rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/license/LicenseController.java

```diff
@@ -526,4 +526,21 @@ public ResponseEntity<RequestStatus> createLicenseType(
         HttpStatus status = HttpStatus.OK;
         return new ResponseEntity<>(requestStatus, status);
     }
+
+    @Operation(
+            summary = "Delete a specific license type.",
+            description = "Delete a specific license type.",
+            tags = {"Licenses"}
+    )
+    @PreAuthorize("hasAuthority('WRITE')")
+    @RequestMapping(value = LICENSES_URL + "/{id}/deleteLicenseType", method = RequestMethod.DELETE)
```

Comment:
Following REST API practices, the request should look like
```suggestion
    @RequestMapping(value = LICENSES_URL + "/{id}", method = RequestMethod.DELETE)
```